<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="system_store">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>851</width>
      <height>631</height>
     </rect>
    </property>
    <property name="title">
     <string>Система магазина</string>
    </property>
    <widget class="QPushButton" name="log_out_button">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Выйти</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>811</width>
       <height>561</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <layout class="QHBoxLayout" name="reports">
        <item>
         <widget class="QCalendarWidget" name="calendarWidget"/>
        </item>
        <item>
         <widget class="QListWidget" name="reports_2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QGroupBox" name="access1">
          <property name="title">
           <string>Уровень доступа 1</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QPushButton" name="sell">
               <property name="text">
                <string>Продать товар</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clean">
               <property name="text">
                <string>Провести уборку</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="plumber">
               <property name="text">
                <string>Провести сантехнические работы</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="electricity">
               <property name="text">
                <string>Проверка электричества</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="add_on_warehouse">
               <property name="text">
                <string>Добавление на склад</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="access2">
          <property name="title">
           <string>Уровень доступа 2</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QPushButton" name="inventory">
               <property name="text">
                <string>Инветаризация</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="check_warehouse">
               <property name="text">
                <string>Переучет склада</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="new_prod">
               <property name="text">
                <string>Закупить новый товар</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="access3">
          <property name="title">
           <string>Уровень доступа 3</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QPushButton" name="worker">
               <property name="text">
                <string>Нанять сотрудника</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buy_equi">
               <property name="text">
                <string>Купить оборудование</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_10">
               <property name="text">
                <string>Уволить сотрудника</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>Список работников</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="sing_in">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>281</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Вход в систему</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>191</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Войдите в систему</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_login">
            <property name="text">
             <string>Логин</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Label_password">
            <property name="text">
             <string>Пароль</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLineEdit" name="login"/>
          </item>
          <item>
           <widget class="QLineEdit" name="password">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="log_in">
        <property name="text">
         <string>Войти</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>sing_in</zorder>
   <zorder>system_store</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>856</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
